code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -456,3 +456,23 @@
 
 /datum/objective/abductee/pragnant
 	explanation_text = "You are pregnant and soon due. Find a safe place to deliver your baby."
+
+/datum/objective/abductee/engine
+	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
+	
+/datum/objective/abductee/teamredisbetterthangreen
+	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
+	
+/datum/objective/abductee/time/New()
+	..()
+	if(istype(ticker.mode, /datum/game_mode/clockwork_cult))
```

Comment:
Cheridan abductors are pulled from ghosts so they will likely know the round type anyway